src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,91 @@
-import React from 'react';
+import {
+  FC,
+  FormEvent,
+  useMemo,
+  useState,
+  memo,
+  ChangeEvent,
+} from 'react';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  movies: Movie[];
+  addMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: FC<Props> = memo(({ movies, addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isIncluded, setIsIncluded] = useState(false);
+
+  // eslint-disable-next-line max-len
+  const DEAFULT_PICTURE = useMemo(() => (
+    'https://via.placeholder.com/360x270.png?text=no%20preview'
+  ), []);
```

Comment:
will be better to move constants outside of the component

Reply:
and you shouldn't cover primitive data types with useMemo
we need useMemo for functions / objects because for each re-rendering corresponding variables will store the new link for some function or object
primitive values doesn't change in this way